bundles/Billing/Service/BillingService.php

```diff
@@ -776,21 +833,32 @@ public function exportInvoicesToSpreadsheet(array $invoiceIds)
             ++$row;
 
             foreach ($invoice->getInvoiceEntries() as $invoiceEntry) {
+                $materialNumber = $invoiceEntry->getMaterialNumber();
+                $product = $invoiceEntry->getProduct();
+                $amount = $invoiceEntry->getAmount();
+                $price = $invoiceEntry->getPrice();
+                $account = $invoiceEntry->getAccount();
+
+                // Ignore lines that have missing data.
+                if (!$materialNumber || !$product || !$amount || !$price || $account) {
+                    continue;
+                }
+
                 // Generate invoice lines (L).
                 // A. "Linietype"
                 $sheet->setCellValueByColumnAndRow(1, $row, 'L');
                 // B. "Materiale (vare)nr.
-                $sheet->setCellValueByColumnAndRow(2, $row, str_pad($invoiceEntry->getMaterialNumber(), 18, '0', STR_PAD_LEFT));
+                $sheet->setCellValueByColumnAndRow(2, $row, str_pad($materialNumber, 18, '0', STR_PAD_LEFT));
                 // C. "Beskrivelse"
-                $sheet->setCellValueByColumnAndRow(3, $row, $invoiceEntry->getProduct());
+                $sheet->setCellValueByColumnAndRow(3, $row, $product);
                 // D. "Ordremængde"
-                $sheet->setCellValueByColumnAndRow(4, $row, number_format($invoiceEntry->getAmount(), 3, ',', ''));
+                $sheet->setCellValueByColumnAndRow(4, $row, number_format($amount, 3, ',', ''));
```

Comment:
Maybe use ”Number formats”: https://phpspreadsheet.readthedocs.io/en/latest/topics/recipes/#number-formats?

Reply:
I will leave this as is atm.